mutiny-core/src/redshift.rs

```diff
@@ -425,12 +427,6 @@ impl RedshiftManager for NodeManager {
             rs.status
         );
 
-        // fixme, this could close the wrong channel if the receiving_node already had channels
```

Comment:
ah shit @TonyGiorgio this is still the case after your PR

Reply:
No, the close channel flow only happens if it's an on chain redshift. So there is no chance that a receiving node already had channels.

Reply:
Ah I see, so we'll never set that channel outpoint then. Probably okay for now

Reply:
Yeah, I added this comment for the outpoint itself: 

```
    /// output_channel will be None if being kept in lightning, this is only relevant when the
    /// channels will be closed into an on chain address
```

Realized it isn't really something that matters for the lightning option (and as we determined by matt, a gibberish thing for now that's not possible to determine the channel payments came down)